aqueduct/lib/src/db/postgresql/query_builder.dart

```diff
@@ -33,33 +35,35 @@ class PostgresQueryBuilder extends TableBuilder {
 
   bool get containsJoins => returning.reversed.any((p) => p is TableBuilder);
 
-  String get sqlWhereClause {
+  String? get sqlWhereClause {
     if (predicate?.format == null) {
       return null;
     }
-    if (predicate.format.isEmpty) {
+    if (predicate!.format!.isEmpty) {
       return null;
     }
-    return predicate.format;
+    return predicate?.format;
   }
 
   void addColumnValueBuilder(String key, dynamic value) {
     final builder = _createColumnValueBuilder(key, value);
-    columnValueBuildersByKey[builder.sqlColumnName()] = builder;
-    variables[builder.sqlColumnName(withPrefix: valueKeyPrefix)] =
-        builder.value;
+    if (builder != null) {
```

Comment:
Opted for noop here, also considered catching a null check error and printing a warning.